### PR TITLE
[MINOR] Add copy param testcase to_numpy in indexes

### PR DESCRIPTION
**Pull request by beobest2**
image

It's a small part, but I've added it to improve code coverage.

### CODE REVIEW

File: databricks/koalas/indexes.py [hunk 1]
```diff
@@ -386,6 +386,8 @@ def to_numpy(self, dtype=None, copy=False):
         array([0, 1, 2, 3])
         >>> ks.DataFrame({'a': ['a', 'b', 'c']}, index=[[1, 2, 3], [4, 5, 6]]).index.to_numpy()
         array([(1, 4), (2, 5), (3, 6)], dtype=object)
+        >>> ks.DataFrame({'a': ['a', 'b', 'c']}, index=[[1, 2, 3], [4, 5, 6]]).index.to_numpy(copy=True)
```
Comment (itholic) [inline databricks/koalas/indexes.py]: too long here. (104 > 100)

**Comment by itholic** [comment]
Otherwise, seems fine.

File: databricks/koalas/indexes.py (hunk 1 above)
Comment (itholic) [reply databricks/koalas/indexes.py]: FYI: You can run a linter-check before push the commit via `/dev/lint-python`

Comment (HyukjinKwon) [inline databricks/koalas/indexes.py]: This example doesn't look useful enough to show in the docstring and documentation.
Shell we move this test case into `test_indexes`?

Comment (beobest2) [reply databricks/koalas/indexes.py]: @HyukjinKwon okay I will move this test case into `test_indexes`

Comment (beobest2) [reply databricks/koalas/indexes.py]: @itholic sorry I forgot to check lint

**Pull request closed without merging**